plantseg/segmentation/dtws.py

```diff
@@ -1,8 +1,50 @@
+import time
+from functools import partial
+
 import numpy as np
-from GASP.segmentation import WatershedOnDistanceTransformFromAffinities
+from elf.segmentation.watershed import distance_transform_watershed, stacked_watershed
 
+from plantseg.pipeline import gui_logger
 from plantseg.pipeline.steps import AbstractSegmentationStep
-from plantseg.segmentation.utils import shift_affinities
+
+
+def compute_distance_transfrom_watershed(pmaps, threshold, sigma_seeds,
+                                         stacked=False, sigma_weights=2.,
+                                         min_size=100, alpha=1.0,
```

Comment:
From my experience setting alpha to some value smaller 1 can improve results a bit, a good range is `[0.75,0.95]`.

Reply:
Good point @constantinpape, I tested different alphas and with alpha lower than 1 I found the results to be better in 2D and worst in 3D. Could it be due to the anisotropy of the data?

Reply:
Yes, anisotropy might be a factor here. This could be improved by adding the `pixel_pitch`, as @wolny mentioned below.

Reply:
This would require quite a bit refactor because all workflows are voxel size agnostic. I still think it's a good idea, but I need to think a bit how to refactor the code.